staging/src/k8s.io/api/extensions/v1beta1/types.go

```diff
@@ -927,6 +927,10 @@ type PodSecurityPolicySpec struct {
 	// is allowed in the "volumes" field.
 	// +optional
 	AllowedFlexVolumes []AllowedFlexVolume `json:"allowedFlexVolumes,omitempty" protobuf:"bytes,18,rep,name=allowedFlexVolumes"`
+	// AllowedCSIDrivers is a whitelist of allowed CSI drivers.  This slice will be checked
+	// to verify which CSI driver is allowed to run in the cluster.
+	// +optional
+	AllowedCSIDrivers []AllowedCSIDriver `json:"allowedCSIDrivers,omitempty" protobuf:"bytes,23,rep,name=allowedCSIDrivers"`
```

Comment:
I'm unsure about the policy here. Do we want to go back and add new fields to the older (v1beta/alpha) types if we already have a v1 version?

Reply:
See response in newer PR here - https://github.com/kubernetes/kubernetes/pull/74086#issuecomment-464188838